### ISSUE TITLE
AWS Lamba setup

### ISSUE THREAD

**Issue by eschizoid** [mtvg/August#7]
Could you please give more detail on which `*.js` file(s) need to be uploaded to AWS and what files are the ones that need be run locally? Im working on dockerizing this project to make it easier to use for everybody. 

Thanks

Mariano

**Comment by mtvg** [comment]
Hey Mariano, 

This was not meant to work on Lambda, the Alexa system connects directly to your Raspberry Pi.
In the Alexa developer portal, indicate the URL of your endpoint according to the `config.alexaSkillGatewayURL` set in the `config.js` file. (It is set to `/august/alexa` by default)
For example, if your Raspberry Pi public IP is 20.0.0.1, the endpoint URL would be `https://20.0.0.1/august/alexa`

**Comment by eschizoid** [comment]
Hey Mathieu,

Now I completely understand, thanks for the information. Would you be interested on a PR for dockerizing this great project? I'm getting very close to have this up and running with docker and RaspberryPi.

**Comment by mtvg** [comment]
Absolutely, that would make things easier for people who wants to play with this project. Thanks!
